### DISCUSSION TITLE
DKIM import function request

### DISCUSSION THREAD

**Discussion by Lumanet2012** [deeztek/Hermes-Secure-Email-Gateway#37] [Ideas]
Hey Team,

first of NICE NICE work. I am a big supporter of your product and we just done a migration to a new Data Center in Dallas, 
During all this we ran into a situation needing a new E-Mail Gateway Server which we did install and we did configure it all with wildcard SSL and TLS and you name it. 

But we do already have all our DKIM key's and do like to import ours instead of having to generate new keys. So It would be a very nice feature if under "Edit Domain DKIM Configuration" we do get a import like copy past field option so we can import quickly them DKIM keys.

**Comment by deeztek** [comment]
added enhancement issue #38
